languages/rust/exercises/concept/vec-macro/.meta/example.rs

```diff
@@ -0,0 +1,20 @@
+/// vec-macro for creating empty vectors, and vectors of particular size, with vec!
+
+/// Create an empty vector
+fn create_empty() -> Vec<u8> {
+    vec![]
+}
+
+/// Create a buffer of `count` zeroes.
+///
+/// Applications use these for filling and modifying data
+fn create_buffer(count: usize) -> Vec<u8> {
+    vec![0; count]
+}
+
+/// Create a vector containing the first five digits of the Fibonacci sequence.
+///
+/// Fibonacci's sequence is the list of numbers where the next number is a sum of the previous two.
```

Comment:
```suggestion
/// Fibonacci's sequence is the list of numbers where the next number is a sum of the previous two.
/// Its first five elements are `1, 1, 2, 3, 5`.
```

No harm in giving them the values necessary. We're not testing them on sequences.

Reply:
Great point!